README.md

```diff
@@ -1,3 +1,8 @@
 # Confluent Apache Kafka for Developers Course
 
-This is the source code accompanying the [Confluent Apache Kafka for Developers](https://github.com/confluentinc/training-developer) course.
+This is the source code accompanying the **Confluent Apache Kafka for Developers** course.
+
+It is organized in two subfolders `solution` and `labs`. The former contains the complete sample solution for each exercise whilst the latter contains the scaffoliding for each exercise and is meant to be used and elaborated on by the students during the hands-on.
+
+For further information please consult our [Confluent Training](https://confluent.io/training) site.
```

Comment:
Has this been updated? It doesn't seem to make much sense to update it on a website - that seems like a lot of maintenance headaches.